### PR TITLE
rehim_1998_electrochemical_1103

### PR DESCRIPTION
**Pull request by markus-chem**
add lit folder

### CODE REVIEW

File: literature/rehim_1998_electrochemical_1103/rehim_1998_electrochemical_1103_p2_1.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    
+curator:
+    version: 1
+    name: Markus Schilling
+    orcid: https://orcid.org/0000-0003-2684-3711
+    digitized: 2021-10-18 # YYYY-MM-DD
+    
+figure description: #This section will be updated by module `CV`
+    version: 1
+    linked measurements: X-ray diffraction # FTIR, SXRD, Ring Disc - indicate if any other operando methods were shown in the same plot, which is linked to the data
```
Comment (DunklesArchipel) [inline literature/rehim_1998_electrochemical_1103/rehim_1998_electrochemical_1103_p2_1.yaml:20]: XRD?

File: literature/rehim_1998_electrochemical_1103/rehim_1998_electrochemical_1103_p2_1.yaml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                height:
+                    value:
+                    unit:
+                diameter:
+                    value:
+                    unit:
+            geometric electrolyte contact area: 
+                value:
+                unit:
+            preparation procedure: Ag elctrode polished # Short description of sample preparation. This can be very useful. :)
```
Comment (DunklesArchipel) [inline literature/rehim_1998_electrochemical_1103/rehim_1998_electrochemical_1103_p2_1.yaml:102]: Can the polishing be a bit more specific, i.e., electropolishing or mechanical polishing?

**Review by DunklesArchipel — APPROVED** [review]
There are some entries that you might want to change slightly.

File: literature/rehim_1998_electrochemical_1103/rehim_1998_electrochemical_1103_p2_1.yaml (hunk 1 above)
Comment (markus-chem) [reply literature/rehim_1998_electrochemical_1103/rehim_1998_electrochemical_1103_p2_1.yaml:20]: is adapted to XRD

File: literature/rehim_1998_electrochemical_1103/rehim_1998_electrochemical_1103_p2_1.yaml (hunk 2 above)
Comment (markus-chem) [reply literature/rehim_1998_electrochemical_1103/rehim_1998_electrochemical_1103_p2_1.yaml]: is adapted

**Pull request merged**